css/components/badge.css

```diff
@@ -0,0 +1,53 @@
+.badge-container {
+  position: relative;
+}
+.badge {
+  height: 1.5rem;
+  min-width: 1.5rem;
+  padding: 5px;
+  border-radius: 50%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border: 1px solid var(--white-color);
+  color: var(--white-color);
+
+  position: absolute;
+  font-size: 0.8rem;
+}
+
+.badge-green {
+  background-color: var(--success-color);
+}
+.badge-yellow {
+  background-color: var(--warning-color);
+}
+.badge-red {
+  background-color: var(--danger-color);
+}
+.badge-icon-right-top-sm {
+  height: 0.4rem;
+  min-width: 0.4rem;
+  right: -0.4em;
+  top: -1.5em;
+}
+.badge-icon-right-bottom {
+  height: 1rem;
+  min-width: 1rem;
+  left: 3em;
+  bottom: -0.2em;
+  font-size: 0.5rem;
```

Comment:
All good, one suggestion though - px units are better for very small rem values

Reply:
Thank you so much for your suggestion.